<!doctype html><html lang=en-us><head><meta charset=utf-8><title>Genetic Sequence Ambiguity Analyzer - The .aux Archive</title><meta name=author content="Nakul Bhat"><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=stylesheet href=https://nakulbhat.github.io/css/latex.css><link rel=stylesheet href=https://nakulbhat.github.io/css/custom.css><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css><link rel=stylesheet href=https://nakulbhat.github.io/css/main.css><link href=https://unpkg.com/aos@2.3.1/dist/aos.css rel=stylesheet><script id=MathJax-script async src=https://nakulbhat.github.io/js/tex-mml-chtml.js></script></head><body><header><nav class=navbar><div class=nav><ul class=nav-links></ul></div></nav><div class=intro-header><div class=container><div class=posts-heading><h1 data-aos=fade-up data-aos-duration=1000>Genetic Sequence Ambiguity Analyzer</h1></div></div></div></header><div id=content><div class=container role=main><article class=article class=blog-post><div class=postmeta><span class=meta-post><i class="fa fa-calendar-alt"></i>
Sep 4, 2025</span></div><br><p>A simple project to quantify the ambiguity in COVID-19 gene sequences.</p><h2 id=genetic-ambiguity>Genetic Ambiguity</h2><p>In the last two decades, genetic sequencing has become increasingly affordable and accessible. This has led to a rapid growth in sequencing data and the development of platforms dedicated to storing and sharing such data.</p><p>One notable example is the Global Initiative on Sharing All Influenza Data (GISAID). Since its establishment in 2008, GISAID has received over 21 million genetic sequence submissions
<label for=8d5a42dedc19c7a5613e1a05d9a8afa4 class="sidenote-toggle sidenote-number"></label><input type=checkbox id=8d5a42dedc19c7a5613e1a05d9a8afa4 class=sidenote-toggle>
<span class=sidenote><a href=https://gisaid.org>The GISAID Project</a>. Accessed: Sep 2025.</span>
.</p><h2 id=quantifying-genetic-ambiguity>Quantifying Genetic Ambiguity</h2><p>The scale of data available presents challenges in identifying reliable sequences. Under the supervision of Dr. Balaji S, Department of Biotechnology, I developed a method to quantify the quality of genetic sequences.</p><p>This method involves analyzing ambiguity bases (e.g., N, B, H) and their frequency within codon reading frames. By localizing ambiguity to functional regions, we can assess potential damage or unreliability in samples.</p><p>The project is ongoing and collaborations are welcome. Further details are available upon request. Please see my contact page for more details.</p><h3 id=what-ive-learnt>What Iâ€™ve Learnt</h3><p>This project was my initial exposure to bioinformatics and Python programming. It helped me build essential skills such as data preprocessing, feature extraction, and pattern recognition.</p><p>Although results are yet to be published, this project served as a foundation for subsequent work in bioinformatics.</p><div class=blog-tags><a href=https://nakulbhat.github.io/tags/projects/>Projects</a>&nbsp;</div></article></div></div><footer><div class=container><p class="credits copyright"><p class="credits theme-by"><a href=https://nakulbhat.github.io/>The .aux Archive</a>,
&copy;
2025
<a href=https://nakulbhat.github.io/about/>Nakul Bhat</a></p></div></footer><script src=https://unpkg.com/aos@2.3.1/dist/aos.js></script><script>AOS.init()</script></body></html>